.travis.yml

```diff
@@ -7,12 +7,13 @@ services:
   - docker
 before_install:
   - cat /proc/cpuinfo
-  - docker build . --tag ldbc/datagen
+  - docker build . -t ldbc/datagen -f spark.dockerfile
```

Comment:
do we need the Hadoop version? If not we could `mv spark.dockerfile Dockerfile` to simplify

Reply:
We don't. Good suggestion.